geopandas/io/tests/test_file.py

```diff
@@ -27,8 +28,13 @@
 try:
     import pyogrio
 
+    PYOGRIO_GE_090 = Version(Version(pyogrio.__version__).base_version) >= Version(
+        "0.9.0"
```

Comment:
Moved this version check here (we have the same for Fiona, also `FIONA_GE_19` is redefined here). Problem is that otherwise at time of import of the module, that variable is not yet properly initialized (which only happens at first time importing pyogrio). 
Will add a comment to clarify that.

Right now this was therefore always skipping the test, even with pyogrio 0.9.0

Reply:
Ta thanks - missed that the fiona logic is here as well and can't be done by import.